AVPlayerExample/ViewController.m

```diff
@@ -195,31 +196,39 @@ - (void)prepareMedia {
     [self startPreview];
 }
 
-- (void)setupAudioSession {
-    // In this example we don't want TwilioVideo to dynamically configure and activate / deactivate the AVAudioSession.
-    // Instead we will setup audio once, and deal with activation and de-activation manually.
-    [[TVIAudioController sharedController] configureAudioSession:TVIAudioOutputVideoChatDefault];
+- (void)startAudioDevice {
+    if (!self.audioDevice) {
+        self.audioDevice = [TVIDefaultAudioDevice audioDeviceWithBlock:^{
```

Comment:
Since this is sample code, can you add a comment describing what is being done here? You can reuse portions of the original.